automation/alpha-archived-metadata-file-upload-check.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VAULT_TOKEN=$(cat /etc/vault-token-dsde)
+
+# Get a workflow which completed at least 40 minutes ago and whose metadata is archived
+# (archive-delay in alpha is set to 30 minutes. Hence adding 10 more minutes should give Cromwell enough time to
+# archive such a workflow)
+ARCHIVED_WORKFLOW_RESULT=$(
+  docker run --rm \
+     -e VAULT_TOKEN=$VAULT_TOKEN \
+     broadinstitute/dsde-toolbox mysql-connect.sh -p firecloud -a cromwell1 -e alpha \
+     "SELECT WORKFLOW_EXECUTION_UUID, ROOT_WORKFLOW_EXECUTION_UUID FROM cromwell.WORKFLOW_METADATA_SUMMARY_ENTRY WHERE METADATA_ARCHIVE_STATUS = 'Archived' AND END_TIMESTAMP <= (NOW() - INTERVAL 40 minute) ORDER BY END_TIMESTAMP DESC LIMIT 1;"\
+     | tail -n1
+)
+
+WORKFLOW_ID=$(echo $ARCHIVED_WORKFLOW_RESULT | cut -d' ' -f1)
+ROOT_WORKFLOW_ID=$(echo $ARCHIVED_WORKFLOW_RESULT | cut -d' ' -f2)
```

Comment:
Is it worth having a second instance of this test which deliberately picks an archived subworkflow to guarantee that it was put into its root workflow's directory?

Reply:
I think there's no harm in having additional check for that. The only thing I am concerned about is whether we have workflows run in alpha on daily basis that have subworkflows (I say daily because of above comment where we eventually want to get a workflow id that was run within the last day). The 35k scatter workflow doesn't have any subworkflow. I will check if there are any such workflows run in alpha at certain intervals, and if they are, I can add additional check for the subworkflows.

Reply:
So I checked in alpha db, and it looks like we don't have any Jenkins tests that have workflows with subworkflows in it. If you think we should check that archived subworkflow is put in its root workflow's directory, then we would have to add a Jenkins test that runs a workflow with subworkflow in it.

Reply:
Maybe we should write this up as a separate ticket and decide in a refinement whether it's worth doing or not? Then for now, it can not a blocker to completing this ticket.

Reply:
Created https://broadworkbench.atlassian.net/browse/BW-658